app/pods/components/publications/publication/activity-request-panel/component.js

```diff
@@ -1,9 +1,45 @@
 import Component from '@glimmer/component';
 import { tracked } from '@glimmer/tracking';
 import { action } from '@ember/object';
+import CONFIG from 'fe-redpencil/utils/config';
+import { inject as service } from '@ember/service';
+
+/*
+*  @activity={{activity}}
+*  @finishedToggle: function
+*  @cancelExisting: function
+*  @translation: true/false
+ */
 
 export default class ActivityRequestPanel extends Component {
+  @service store;
   @tracked isCollapsed = false;
+  @tracked checked;
+
+  get mode() {
+    if (this.args.mode === 'translation') {
+      return true;
+    }
+    return false;
```

Comment:
Dit kunnen we one-linen als ik me niet vergis `return this.args.mode === 'translation'` en ik zou de getter dan `isTranslation` noemen bijvoorbeeld.